server/server.go

```diff
@@ -1226,7 +1226,33 @@ func (a *ArgoCDServer) getClaims(ctx context.Context) (jwt.Claims, string, error
 	if err != nil {
 		return claims, "", status.Errorf(codes.Unauthenticated, "invalid session: %v", err)
 	}
-	return claims, newToken, nil
+
+	// Some SSO implementations (Okta) require a call to
+	// the OIDC user info path to get attributes like groups
+	// we assume that everywhere in argocd jwt.MapClaims is used as type for interface jwt.Claims
+	// otherwise this would cause a panic
+	var groupClaims jwt.MapClaims
+	if groupClaims, ok = claims.(jwt.MapClaims); !ok {
+		groupClaims = *claims.(*jwt.MapClaims)
```

Comment:
I removed the second type-conversion inside the if-statement and replaced it so that it's now catched. Is that what you meant? I actually forgot why I had to do this second type-conversion and I can also no longer reproduce it.